tests/foreman/api/test_activationkey.py

```diff
@@ -55,476 +55,466 @@ def _bad_max_hosts():
     return [gen_integer(-100, -1), 0, gen_string('alpha')]
 
 
-class ActivationKeyTestCase(APITestCase):
-    """Tests for the ``activation_keys`` path."""
+@tier1
```

Comment:
As #8092 has been merged, you might want to make the new changes? `@pytest.mark.tier1`

Reply:
I also see there are many such cases and I rather not flood the review section. Please make those changes as well. Thanks.

Reply:
Make sure you add the docstring `parametrized: yes` to each test cases that are parametrized as well.